Tasks/VsTest/task.json

```diff
@@ -3,21 +3,22 @@
     "name": "VSTest",
     "friendlyName": "Visual Studio Test",
     "description": "Run tests with Visual Studio test runner",
-    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=624539)",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=835764)",
```

Comment:
Added new fwdlink, which still points to old documentation. Prachi to update the fwdlink with documentation for preview version.